operator/pkg/test_samples/oci/config.yaml

```diff
@@ -0,0 +1,7 @@
+# Samples Config
+configs:
```

Comment:
Added by accident?

Reply:
This has been added as part of the layer.Uncompressed() part in `manifest_controller_test.go`